Backend/app/rider-platform/rider-app/Main/src/SharedLogic/CallBPP.hs

```diff
@@ -316,6 +276,31 @@ callBecknAPIWithSignature ::
   m res
 callBecknAPIWithSignature a = callBecknAPI (Just $ Euler.ManagerSelector $ getHttpManagerKey a) Nothing
 
+callBecknAPIWithSignature' ::
+  ( MonadFlow m,
+    CoreMetrics m,
+    IsBecknAPI api req res,
+    SanitizedUrl api,
+    ToJSON req,
+    CacheFlow m r,
+    HasFlowEnv m r '["kafkaProducerTools" ::: KafkaProducerTools],
+    EsqDBFlow m r
+  ) =>
+  Id Merchant.Merchant ->
+  VehicleCategory ->
+  Text ->
+  Text ->
+  Proxy api ->
+  BaseUrl ->
+  HM.HashMap BaseUrl BaseUrl ->
+  req ->
+  m res
+callBecknAPIWithSignature' merchantId vehicleCategory a b c d e req' = do
```

Comment:
You can pass `becknConfig` instead of `merchantId`, since its anyways looked up in the business logic